packages/amplify-cli/src/commands/configure.ts

```diff
@@ -15,12 +16,25 @@ export const run = async (context: Context) => {
     context.print.success('Usage Data has been turned off');
     return;
   }
-  if (context.parameters.options['usage-data-on']) {
+
+  if (context.parameters.options['usage-data-on ']) {
     write(context, { usageDataConfig: { isUsageTrackingEnabled: true } });
     context.print.success('Usage Data has been turned on');
     return;
   }
 
+  if (context.parameters.options['share-project-config-off']) {
+    DebugConfig.Instance.setAndWriteShareProject(false);
+    context.print.success('Share Project Config has been turned off');
```

Comment:
context.print operations are deprecated consider using `printer` from the `@aws-amplify/amplify-prompts` package, but not a merge blocker.

Reply:
Done 3a7220a335dc8aa1e4243982f973137ddd616726